dbt/adapters/athena/impl.py

```diff
@@ -61,12 +62,32 @@
 boto3_client_lock = Lock()
 
 
+@dataclass
+class AthenaConfig(AdapterConfig):
```

Comment:
nice 💯

Reply:
if I understand correct, this class should contain all additional model configs the adapter work with. If it's so, then we should add following configs:
```
seed_s3_upload_args: Otional[Dict[str, Any]] = None  # https://github.com/dbt-athena/dbt-athena/commit/9d029bbfd1fdab2c8c7f9ebe7185e7a34f4be63e
partitions_limit: Optinal[int] = None  #https://github.com/gr8-toolkit/dbt-athena/blob/4394dfa3abd20c8e87345de3f0a271582d56ebcb/dbt/include/athena/macros/materializations/models/helpers/get_partition_batches.sql#L3-L3
```

Reply:
also It would be perfect to add docstring with these parameters description

Reply:
do we support partitions_limit globally? to me seems more a model config, e.g. [here](https://github.com/dbt-athena/dbt-athena/blob/9dec391257fce15b1887d8fa6bb8dd4b7e8261a2/dbt/include/athena/macros/materializations/models/helpers/get_partition_batches.sql#L3)

Might not be a bad idea to do something like this
```
{%- set athena_partitions_limit = config.get('partitions_limit', target.partitions_limit) | int -%}
```

Reply:
> do we support partitions_limit globally?
now not, but such config can be added of course